guides/welfare/raising_an_issue.md

```diff
@@ -28,9 +22,9 @@ You can also call the [Acas](http://www.acas.org.uk/index.aspx?articleid=2042) o
 
 At the time of writing we do not have anyone with mediation training at the company but if necessary we can bring in third party mediators to deal with disputes informally.
 
-You an find out more about Mediation from [Acas](http://www.acas.org.uk/index.aspx?articleid=1680).
+You can find out more about Mediation from [Acas](http://www.acas.org.uk/index.aspx?articleid=1680).
 
-If you wish to enter a mediation process, please ask a director or ask a colleague to go to a director to request mediation.
+If you wish to enter a mediation process, please ask the people team to request mediation.
```

Comment:
What happens if you have a problem with the people team?

Reply:
CEO has ultimate responsibility in that case and would be required to step in to mediate I imagine.

Reply:
I believe there is an obligation to enter into mediation, and a neutral mediator will be provided. That should not affect which team requests mediation.